modules/lib/file-type.nix

```diff
@@ -95,6 +95,14 @@ in
             are symbolic links to the files of the source directory.
           '';
         };
+
+        onChange = mkOption {
+          type = types.nullOr types.lines;
+          default = null;
```

Comment:
I think it would be nicer to have `type = types.lines; default = ""` and then filter out empty strings rather than null values.